sway-core/src/transform/to_parsed_lang/convert_parse_tree.rs

```diff
@@ -279,6 +283,7 @@ fn use_tree_to_use_statements(
             };
             ret.push(UseStatement {
                 call_path: path.clone(),
+                span: item_span.clone(),
```

Comment:
The `.clone()` is not needed here

Reply:
Good catch, I'm surprised clippy didn't find these!